proxy/envoy/policy.go

```diff
@@ -61,6 +61,7 @@ func insertMixerFilter(listeners []*Listener, instances []*model.ServiceInstance
 							"source.ip":  context.IPAddress,
 							"source.uid": context.UID,
 						},
+						QuotaName: "RequestCount",
```

Comment:
So due to code changes in mixer it's no longer trivial to move back to `Check` based quota; we could write some shims to do it, but it's not as easy as adding the quota aspect manager into the check manager set unfortunately.